lwt/disk.ml

```diff
@@ -46,11 +47,8 @@ let start_upload ~chunked ~uri =
     | Some "https" -> true
     | Some "http" -> false
     | x -> failwith (Printf.sprintf "Unsupported URI scheme: %s" (match x with None -> "None" | Some x -> x)) in
```

Comment:
`use_ssl` is also duplicated, you could add an `is_ssl : url -> bool` function to util.ml too

Reply:
indeed

Reply:
I lost some info in the refactor, should i had a `allow_file` or something like that when i want only http(s)?

Reply:
Would that work?
```ocaml
let is_ssl uri allow_file =
  match Uri.scheme uri with
  | Some "https" ->
    true
  | Some "http"
  | Some "file" when allow_file ->
    false
  | x ->
    failwith (Printf.sprintf "Unsupported URI scheme: %s" (match x with None -> "None" | Some x -> x))
```